test/integration/spec/rtcpeerconnection.js

```diff
@@ -861,7 +867,9 @@ function testClose(sdpSemantics, signalingState) {
         test.peerConnection.addEventListener('signalingstatechange', onSigalingStateChanged);
         var closePromise = test.close();
         test.peerConnection.removeEventListener('signalingstatechange', onSigalingStateChanged);
-        return closePromise.then(results => result = results[0]);
+        return closePromise.then(results => {
+          result = results[0]
+        });
```

Comment:
Can you please revert this, or, if lint-ing requires the `{`/`}`, at least keep on one line?